setup.py

```diff
@@ -66,6 +66,7 @@ def run(self):
 		'aiohttp>=3.6.2',
 		'fastjsonschema>=2.14.4',
 		'kazoo>=2.8.0',
+		'pygit2>=1.9.1'
```

Comment:
This should be optional dependency, only if the GitProvider is used (imported) then this should be needed.

Reply:
When installing ASAB, how do I know whether I (or rather ASAB in Docker) will import GitProvider (in some far far future from the installation point of view)?

Reply:
People who wants to us GitProvider in Docker, they will install this one explicitly in Dockerfile ...